CompanyEmployees.Core.Domain/Repositories/IAttendanceRepository.cs

```diff
@@ -0,0 +1,13 @@
+using CompanyEmployees.Core.Domain.Entities;
+
+namespace CompanyEmployees.Core.Domain.Repositories;
+
+public interface IAttendanceRepository
+{
+    Task<IEnumerable<Attendance>> GetAllAttendancesAsync(bool trackChanges, CancellationToken ct = default);
+    Task<Attendance> GetAttendanceAsync(Guid attendanceId, bool trackChanges, CancellationToken ct = default);
+    Task<IEnumerable<Attendance>> GetAttendancesByEmployeeIdAsync(Guid employeeId, bool trackChanges, CancellationToken ct = default);
+    Task<Attendance?> GetTodayAttendanceByEmployeeIdAsync(Guid employeeId, bool trackChanges, CancellationToken ct = default);
+    void CreateAttendance(Attendance attendance);
+    void DeleteAttendance(Attendance attendance);
+}
```

Comment:
Adding interface-level XML documentation comments can improve developer understanding and downstream tool support (e.g. IntelliSense). Consider briefly documenting the purpose of each method.

**Suggested change**
```suggestion
/// <summary>
/// Retrieves all attendance records.
/// </summary>
Task<IEnumerable<Attendance>> GetAllAttendancesAsync(bool trackChanges, CancellationToken ct = default);
```